tests/v2/validation/rbac/rbac_test.go

```diff
@@ -0,0 +1,239 @@
+	//assert cluster list contains right cluster ID
+	if err == nil{
+		actualResult := len(clusterList.Data)
+		assert.Equal(rb.T(), expectedClusterCount, actualResult)
+		if actualResult > 0 {
+			if clusterID != "" {
+				assert.Equal(rb.T(), clusterID, clusterList.Data[0].Status.ClusterName )
+			}
+		}
+	}
+	return err
+}
+
+
+func (rb *rbTestSuite) ValidateClusterMembers(role string) {
+
+	//Testcase1 Verify if cluster members are able to list clusters
+	fmt.Printf("Testcase1 - Validating the cluster count obtained as a %s. \n",role)
+	err := rb.ValidateClustersList(rb.standardUserClient, 1, rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	fmt.Println("Validation successful!")
+
+	//Testcase2 Verify if cluster members are able to list the projects in a cluster
+	fmt.Println("Testcase2 - Validating cluster members are able to list all projects")
+	//Create a project as an admin
+	createProjectAsAdmin, err := CreateProject(rb.client, rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	fmt.Println("Created project as an admin: ",createProjectAsAdmin.Name)
+	//Get project list as an admin
+	projectlistAdmin, err := GetListProjects(rb.client, rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	//Get project list as a cluster owner/member
+	projectlistClusterMembers, err := GetListProjects(rb.standardUserClient, rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	if role == "cluster-owner"{
+		//assert length of projects list obtained as an admin to the cluster owner is equal
+		assert.Equal(rb.T(), len(projectlistAdmin), len(projectlistClusterMembers))
+		//assert projects list obtained as an admin to the cluster owner is the same
+		assert.Equal(rb.T(), projectlistAdmin, projectlistClusterMembers)
+		fmt.Println("Validation successful!")
+	}else if role == "cluster-member" {
+		//assert projects list obtained as a cluster member is empty
+		assert.Equal(rb.T(), 0, len(projectlistClusterMembers))
+		fmt.Println("Validation successful!")
+	}
+
+	//Testcase3 Validate if cluster members can create a project in the downstream cluster
+	fmt.Printf("Testcase3 - Validating if %s is able to create a project in the cluster. \n", role)
+	createProjectAsClusterMembers, err := CreateProject(rb.client, rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	fmt.Printf("Created project as a %s: %s. \n",role,createProjectAsClusterMembers.Name)
+	require.NoError(rb.T(), err)
+	actualStatus := fmt.Sprintf("%v", createProjectAsAdmin.State)
+	assert.Equal(rb.T(),"active",actualStatus)
+	fmt.Println("Validation successful!")
+
+	//Testcase4 Validate if cluster members can create namespaces in project they are not owner of
+	fmt.Printf("Testcase4 - Validating if %s can create namespace in a project they are not owner of. \n",role)
+	namespaceName := AppendRandomString("testns-")	
+	createdNamespace, err := namespaces.CreateNamespace(rb.standardUserClient, namespaceName, "{}", map[string]string{}, map[string]string{}, createProjectAsAdmin)
+	if role == "cluster-owner"{
```

Comment:
is there a 3rd or more cases we may add here that we should catch in an `else` statement? it looks strange to have `if` and `if else` but no `else`.

Reply:
We only have 2 roles for cluster-owner and cluster-member so either if this role or that role.